src/bp/nlu-core/entities/custom-entity-extractor.ts

```diff
@@ -191,7 +191,7 @@ export const extractListEntitiesWithCache = (
   utterance: Utterance,
   list_entities: WarmedListEntityModel[]
 ): EntityExtractionResult[] => {
-  const cacheKey = utterance.toString({ lowerCase: true })
+  const cacheKey = utterance.toString({ lowerCase: true, entities: 'keep-value' })
```

Comment:
oh ... does that mean that the entity cache was simply not working ?

Reply:
+ that doesn't fix the entities cache dump order right ?

Reply:
its a leftover, srry

This solves nothing... I first thought this was the bug I was observing, but it has nothing to do with it. The entities are tagged after the custom-entity-extractor anyway so this changes nothing.

\+ at predict time we don't even use the cache.

Reply:
removed the "keep-value" but added a comment.